repository.py

```diff
@@ -38,11 +39,12 @@ def create_queue(self, queue_name):
             return "INTEGRITY_ERROR"
 
     def add_me_to_queue(self, name, queue_id):
+        crossed = 0
         try:
             self.cursor.execute("""
-                INSERT INTO pupils (name, queue_id)
-                VALUES (?, ?)
-            """, (name, queue_id))
+                INSERT INTO queue_members (name, queue_id, crossed)
```

Comment:
We don't have to edit this function, since the field is going to get initialized to `0` by default